assets/js/base/components/payment-methods/test/payment-methods.js

```diff
@@ -53,10 +53,12 @@ describe( 'PaymentMethods', () => {
 		);
 
 		await waitFor( () => {
-			const noPaymentMethods = screen.queryByText(
+			const noPaymentMethods = screen.queryAllByText(
 				/no payment methods available/
 			);
-			expect( noPaymentMethods ).not.toBeNull();
+			// We might get more than one match because the `speak()` function
+			// creates an extra `div` with the notice contents used for a11y.
+			expect( noPaymentMethods.length ).toBeGreaterThanOrEqual( 1 );
```

Comment:
Recent versions of the `Notice` component generate two elements with the notice text: one for the notice itself and another one used by the `speak()` a11y function. Our previous test was failing because it was expecting one and only one match.